src/DashboardService.Data/Interfaces/IPriorityRepository.cs

```diff
@@ -12,8 +13,10 @@ namespace LT.DigitalOffice.DashboardService.Data.Interfaces;
 public interface IPriorityRepository
 {
   Task<Guid?> CreateAsync(DbPriority dbDepartment);
-  Task<List<DbPriority>> GetAllAsync(GetPrioritiesFilter filter);
-  Task<DbPriority> GetAsync(Guid id);
-  Task<bool> EditAsync(Guid id, JsonPatchDocument<DbPriority> request);
-  Task<bool> RemoveAsync(Guid id);
+  Task<(List<DbPriority> dbPriorities, int totalCount)> GetAllAsync(GetPrioritiesFilter filter, CancellationToken ct);
+  Task<DbPriority> GetAsync(Guid id, CancellationToken ct = default);
+  Task<bool> EditAsync(Guid id, JsonPatchDocument<DbPriority> request, CancellationToken ct);
+  Task<bool> RemoveAsync(Guid id, CancellationToken ct);
+  Task<bool> NameExistAsync(string name);
```

Comment:
May be add CancellationToken to all async methods?

Reply:
to do this i need to use at least ExtendedEditRequestValidator for pass ct to validator constructor. Is it necessary?

Reply:
++